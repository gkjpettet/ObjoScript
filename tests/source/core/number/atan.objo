System.print(0.atan) # Expect 
System.print(1.atan) # Expect 
System.print(0.5.atan) # Expect 