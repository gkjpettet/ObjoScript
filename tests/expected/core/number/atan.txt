0
0.785398163
0.463647609